File: Java/Katas/src/Katas/FizzBuzz/FizzBuzz.java
package Katas.FizzBuzz;

public class FizzBuzz {

    public String toFizzBuzz(int number){

        var result = "";
        var numberToString = Integer.toString(number);

        if (number == 0) return numberToString;
        
        if (number % 3 == 0) result = "fizz";
        if (number % 5 == 0) result = result + "buzz";

        return result;
    }
}

File: PHP/classes/Dojos/FizzBuzz.php
<?php


namespace CodingDojo\Dojos;


class FizzBuzz
{
    public function toFizzBuzz(int $number)
    {
        $result = "";
        $numberToString = (string)$number;

        if ($number == 0) return $numberToString;

        if ($number % 3 == 0) $result = "Fizz";
        if ($number % 5 == 0) $result = $result . "Buzz";

        if (empty($result)) return $numberToString;
        return $result;
    }
}